Master-OB-OpenAPI.json

```
@@ -13129,6 +13132,29 @@
             "x-ob-item-type-group": ""
           }
         ]
+      },
+      "PrivateKeyAddress": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementString"
+          },
+          {
+            "type": "object",
+            "description": "The purpose of this concept is to provide a cryptographically secure identity for a device. The Private Key Address (PKA) is derived from an asymmetric private/public key pair, typically generated on the device itself and stored securely, preferably in a hardware-secured environment like a Secure Element (SE). Private keys remain within this secure environment to prevent exposure. The PKA consists of a type identifier, the hash output of public-key-material, and optionally, a CRC check for error detection, ensuring uniqueness and integrity.",
```

```suggestion
            "description": "This concept provides a cryptographically secure identity for a device. The Private Key Address (PKA) is derived from an asymmetric private/public key pair, typically generated on the device itself and stored securely, preferably in a hardware-secured environment like a Secure Element (SE). Private keys remain within this secure environment to prevent exposure. The PKA consists of a type identifier, the hash output of public-key-material, and optionally, a CRC check for error detection, ensuring uniqueness and integrity.",
```

I'm confused by the last sentence - `PrivateKeyAddress` has a single value, how does it consist of two items (type identifier and hash ouput) and one optional item (CRC check)?